src/content/governance/index.md

```diff
@@ -99,9 +99,9 @@ The initial draft of an EIP is unlikely to be implemented on the Ethereum Mainne
 
 ### Community consensus {#community-consensus}
 
-While some EIPs are straightforward technical improvements with minimal nuance, some are more complex and come with tradeoffs which will affect different stakeholders in different ways. This means some EIPs end up being more contentious within the community than others.
+While some EIPs are straightforward technical improvements with minimal nuance, some are more complex and come with tradeoffs which will affect different stakeholders in different ways. This means some EIPs end up being more contentious within the community than others. In such cases, there will always be various interest groups attempting to control the network through a series of coordination games where widespread support across stakeholder groups is crucial.
```

Comment:
```suggestion
While some EIPs are straightforward technical improvements with minimal nuance, some are more complex and come with tradeoffs which will affect different stakeholders in different ways. This means some EIPs are more contentious within the community than others. In such cases, there will always be various interest groups attempting to control the network through a series of coordination games where widespread support across stakeholder groups is crucial.
```

Reply:
> In such cases, there will always be various interest groups attempting to control the network through a series of coordination games where widespread support across stakeholder groups is crucial

This seems a little jargony / hard to understand.

Reply:
All the regurgitated jargon ;)

It was more to make explicit the coordination required between different categories of stakeholders. Doesn't affect the overall message if the last sentence was to be removed.

Reply:
I think we can remove the last sentence and call this good